build error "‘path’ is not a member of ‘std::filesystem’"

I am trying to upgrade prepair in a project and use v2.0.2, but the build fails.

The full stack:

```
[ 50%] Building CXX object CMakeFiles/prepair.dir/prepair.cpp.o
/tmp/prepair-2.0.2/prepair.cpp: In function ‘int main(int, const char**)’:
/tmp/prepair-2.0.2/prepair.cpp:109:22: error: ‘path’ is not a member of ‘std::filesystem’
  109 |     std::filesystem::path extension = std::filesystem::path(vm["ogrout"].as<std::string>()).extension();
      |                      ^~~~
/tmp/prepair-2.0.2/prepair.cpp:111:9: error: ‘extension’ was not declared in this scope; did you mean ‘boost::fusion::extension’?
  111 |     if (extension.compare(".csv") == 0) out_driver_name = "CSV";
      |         ^~~~~~~~~
      |         boost::fusion::extension
In file included from /usr/include/boost/fusion/adapted/boost_tuple.hpp:12,
                 from /usr/include/boost/iterator/zip_iterator.hpp:20,
                 from /usr/local/include/CGAL/Triangulation_2.h:46,
                 from /usr/local/include/CGAL/Constrained_triangulation_2.h:26,
                 from /usr/local/include/CGAL/Constrained_Delaunay_triangulation_2.h:20,
                 from /tmp/prepair-2.0.2/Polygon_repair.h:26,
                 from /tmp/prepair-2.0.2/prepair.cpp:26:
/usr/include/boost/fusion/adapted/boost_tuple/detail/is_view_impl.hpp:17:15: note: ‘boost::fusion::extension’ declared here
   17 |     namespace extension
      |               ^~~~~~~~~
/tmp/prepair-2.0.2/prepair.cpp:129:26: error: ‘exists’ is not a member of ‘std::filesystem’
  129 |     if (std::filesystem::exists(std::filesystem::path(vm["ogrout"].as<std::string>()))) {
      |                          ^~~~~~
/tmp/prepair-2.0.2/prepair.cpp:129:50: error: ‘path’ is not a member of ‘std::filesystem’
  129 |     if (std::filesystem::exists(std::filesystem::path(vm["ogrout"].as<std::string>()))) {
      |                                                  ^~~~
make[2]: *** [CMakeFiles/prepair.dir/build.make:76 : CMakeFiles/prepair.dir/prepair.cpp.o] Erreur 1
make[1]: *** [CMakeFiles/Makefile2:83 : CMakeFiles/prepair.dir/all] Erreur 2
make: *** [Makefile:136 : all] Erreur 2
```

It looks like a cpp version problem or something like that. gcc is quite recent on my system:

```
$ gcc --version
gcc (Ubuntu 13.3.0-6ubuntu2~24.04) 13.3.0
```

I have this error both on my system (ubuntu 24.04) and in a alpine 3.21 based docker image.

--- a/prepair.cpp
+++ b/prepair.cpp
@@ -19,6 +19,7 @@
 // You should have received a copy of the GNU General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
+#include <filesystem>
 #include <iostream>
 #include <fstream>
 #include <boost/program_options.hpp>

